trace/trace_opentelemetry/main.go

```diff
@@ -0,0 +1,39 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START trace_opentelemetry]
+
+func main() {
```

Comment:
Please run `goimports -w .`. Your editor should be able to be configured to run this automatically. That will automatically format your code and add imports.

Note that if there are new OpenTelemetry packages that you're using, you'll want to run this from inside this directory:

```
go get go.opentelemetry.io/otel
go mod tidy
```

Here are instructions for installing `goimports` manually if your editor doesn't want to work: https://pkg.go.dev/golang.org/x/tools/cmd/goimports?tab=doc